syntax = "proto3";

package Part_Identifier;
option go_package = "./PartIdentifierpb";

message IdentifyPartRequest {
    string part_id = 1;
}

message IdentifyPartResponse {
    string id = 1;
    string title = 2;
    string image_url = 3;
    string part_number = 4; 
    string location = 5; 
}



service PartIdentifierService {
    rpc IdentifyPart(IdentifyPartRequest) returns (IdentifyPartResponse) {};
}